src/wp-includes/theme.json

```diff
@@ -1,6 +1,9 @@
 {
 	"version": 1,
 	"settings": {
+		"border": {
+			"customRadius": false
```

Comment:
Could we enable the setting by default to all blocks?

Reply:
I copied this setting from the plugin. So I'm not sure if we want to enable it for all blocks by default.